python/ray/serve/schema.py

```diff
@@ -186,6 +186,62 @@ def __eq__(self, other: Any) -> bool:
         return self._compute_hash() == other._compute_hash()
 
 
+@PublicAPI(stability="alpha")
+class TracingConfig(BaseModel):
+    """Tracing config for configuring serve tracing."""
+
+    class Config:
+        extra = Extra.forbid
+
+    export_path: str = Field(
+        default=None,
+        description=(
+            "An export path to export traces. Should be of the "
+            'form "module.submodule_1...submodule_n.'
+            'export_tracing". This is equivalent to '
+            '"from module.submodule_1...submodule_n import export_tracing.'
```

Comment:
This needs to describe what the thing that's being imported actually is. From this docstring, users should know how to enable tracing.